Source/WebCore/css/parser/CSSPropertyParserConsumer+URL.h

```diff
@@ -25,6 +25,7 @@
 #pragma once
 
 #include <wtf/Forward.h>
+#include <wtf/OptionSet.h>
```

Comment:
I'm a little surprised that a forward declaration of `OptionSet` is not sufficient and already present in `Forward.h`.